CMakeLists.txt

```diff
@@ -126,8 +126,17 @@ endif()
 add_library(LibJuice::LibJuice ALIAS juice)
 add_library(LibJuice::LibJuiceStatic ALIAS juice-static)
 
+include(GenerateExportHeader)
```

Comment:
The [doc](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html) says: 
> New in version 3.12: Added support for C projects. Previous versions supported C++ project only.

Does this means the change breaks compatibility with CMake before 3.12? (the current `cmake_minimum_required` is 3.7)

Reply:
Right, I forgot to check the required version, sorry. Since the original definitions are going to be kept, I added a condition to enable the new behavior only on CMake >= 12.